quicktask/app/Http/Controllers/Task_controller.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class Task_controller extends Controller
```

Comment:
```suggestion
class TaskController extends Controller
```
Đọc lại code convention (tat)